extensions/azure-openai-oauth-rshiny/manifest.json

```diff
@@ -0,0 +1,1862 @@
+{
+  "version": 1,
+  "locale": "en_US",
+  "platform": "4.3.0",
+  "metadata": {
+    "appmode": "shiny",
+    "primary_rmd": null,
+    "primary_html": null,
+    "content_category": null,
+    "has_parameters": false
+  },
+  "extension": {
+    "name": "azure-openai-oauth-rshiny",
+    "title": "R Shiny Azure OpenAI OAuth Integration Example",
+    "description": "An R Shiny app with an Azure OpenAI OAuth Integration that answers questions about the Palmer Penguins dataset",
+    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/azure-openai-oauth-rshiny",
+    "category": "example",
+    "tags": ["r", "shiny"],
+    "minimumConnectVersion": "2025.06.0",
+    "requiredFeatures": ["OAuth Integrations"],
+    "version": "0.0.0"
```

Comment:
Are you looking to release this in a follow-up PR?

Either way you may want to consider adding this extension to the GitHub workflows for the repo so it can be released, gets linted, tested for deployment, etc. The details for this are in the CONTRIBUTING guide, and I'd be happy to answer any questions you have with that process.